httpserver/config.go

```diff
@@ -120,6 +121,7 @@ func NewDefaultConfig(options ...ConfigOption) *Config {
 		},
 		Log: LogConfig{
 			SlowRequestThreshold: config.TimeDuration(defaultSlowRequestThreshold),
+			TimeSlotsThreshold:   config.TimeDuration(defaultSlowRequestThreshold),
```

Comment:
Let's have zero value for`TimeSlotsThreshold` by default. In that case, `SlowRequestThreshold` will be used.